apps/core/src/dashboards/dashboards.controller.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  Controller,
+  Get,
+  Post,
+  Body,
+  Patch,
+  Param,
+  Delete,
+} from "@nestjs/common";
+import { DashboardsService } from "./dashboards.service";
+import { CreateDashboardDto } from "./dto/create-dashboard.dto";
+import { UpdateDashboardDto } from "./dto/update-dashboard.dto";
+
+@Controller("dashboards")
+export class DashboardsController {
+  constructor(private readonly dashboardsService: DashboardsService) {}
+
+  @Post()
+  create(@Body() createDashboardDto: CreateDashboardDto) {
+    return this.dashboardsService.create(createDashboardDto);
+  }
+
+  @Get()
+  findAll() {
+    return this.dashboardsService.findAll();
+  }
+
+  @Get(":id")
+  findOne(@Param("id") id: string) {
+    return this.dashboardsService.findOne(+id);
```

Comment:
Why do we want to parse the `id` into a number?

Reply:
This is generated by the NestJS CLI. It looks like there was just an assumption that ID is a number. We can remove this.